### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by amklass11**
# responsive mobile nav bar was created
1. no linter errors
2. animations effects added 
3. Click hamburger Icon to  view changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,14 +7,20 @@
...
 	<header class="container">
 		<div id="logo">Lassana</div>
-		<nav>
+		<nav class="nae toggle-menu">
 			<button id="hide" type="button"><i class="fas fa-bars"></i></button>
+			<ul class="mobile-menu" id="anchors">
+				<li class="pad"><a href="#works">Portfolio</a></li>
+				<li  class="pad"><a href="#about-myself">About</a></li>
+				<li  class="pad"><a href="#contact">Contact</a></li>
+			  </ul>
```
Comment (Mre55) [inline index.html:23]: Let's try to stick to the Figma design with the `header`. Currently, it's taking more space than we want. Here is your design compared with the Figma design.

| Your Design | Figma Design |
|-----------|--------------|
| image | image |

Kindly, fix this issue.

File: index.html [hunk 2]
```diff
@@ -7,14 +7,20 @@
...
+
 	
 </head>
 
 <body>
 	<header class="container">
 		<div id="logo">Lassana</div>
-		<nav>
+		<nav class="nae toggle-menu">
 			<button id="hide" type="button"><i class="fas fa-bars"></i></button>
```
Comment (Mre55) [inline index.html:18]: As of the project requirement after the user clicks the `hamburger button` it should have to be changed to the close `(X)` button and should be displayed at the right top of the mobile menu screen.

| Your Design | Figma Design |
|-----------|--------------|
| image | image |

Kindly, fix this too.

File: index.html [hunk 3]
```diff
@@ -7,14 +7,20 @@
...
+			<ul class="mobile-menu" id="anchors">
+				<li class="pad"><a href="#works">Portfolio</a></li>
+				<li  class="pad"><a href="#about-myself">About</a></li>
+				<li  class="pad"><a href="#contact">Contact</a></li>
```
Comment (Mre55) [inline index.html:22]: As of this [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#project-requirements) when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Currently, it's not taking the user to the correct part of the page. Kindly, update the `href` with the appropriate `id` names.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @amklass11,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

- [ ] Kindly, update the pull request description by writing a short summary of changes introduced in the pull request and let the code reviewer know what you did in this project. Currently, it's not telling anything about what you did.

_Check the comments under the review._



## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const menBtn = document.querySelector('.nae');
+const navstuff = document.querySelector('.mobile-menu');
+const Closer = document.querySelector('.close-button');
+const rep = document.getElementById('hide');
+
+menBtn.addEventListener('click', () => {
+  menBtn.classList.toggle('navblur');
+  navstuff.classList.toggle('activate');
+  navstuff.classList.toggle('mobile-menu');
+  Closer.classList.toggle('visible');
```
Comment (nicupop729) [inline app.js:10]: Only when a user clicks the X button, the mobile menu should disappear. At the moment, the mobile menu disappears when the user clicks anywhere on the screen.

- [ ] In order to have a nice functional app, kindly, refactor the code in a way that the mobile menu disappears only when the user clicks the "X" button. :100:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️

Hi team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


**Required changes:**

You have submitted a project, that is not running linter checks for JavaScript.
[Here is the link with the correct linters for this milestone](https://github.com/microverseinc/linters-config/tree/master/html-css-js);

- [ ] Kindly, add linters for JavaScript too in your project!


**[HIGHLY RECOMMENDED]**

The industry practice I to use descriptive names for our branches and for our pull requests.
"feature" is not a happy name for either a branch or a pull request because is not descriptive enough.
"Create the Nav-bar toggle" could be a more appropriate name for this pull request.
"nav-bar-toggle" could have been a better name for this particular branch!

- [ ] Good job having the commits at impretaive tense. We should not forget they should be capitalized too. :1st_place_medal: 

- [ ] Please take another close look at this article: [Professional repo rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

**[BONUS]**

Here is for you a nice description of this milestone which you can use as an example of what a good description of a pull request should look like:
image


- [ ] __Please Check for the rest of the changes in the inline coding.__ :smile: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 4 above)
Comment (amklass11) [reply app.js:10]: @nicupop729 Hello sorry are u saying everything is wrong or just the closing button

Comment (nicupop729) [reply app.js:10]: Hello @amklass11!

At the moment, when the user clicks anywhere, the menu disappears.
It should only be when the user clicks the X button.

It is clear?  Please feel free to ask if not. We can talk more on slack. search me there at @Nicoalae Pop

Comment (amklass11) [reply app.js:17]: @nicupop729 Corrections made what do u think

Comment (nicupop729) [reply app.js:17]: If everything is functional, if the requirements were respected and if you feel prepared, please ask for another review. :smile_cat:

Comment (amklass11) [reply app.js:17]: Alright thank you

**Review by smunozmo — APPROVED** [review]
Hi @amklass11, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**